code/controllers/subsystem/jobs.dm

```diff
@@ -451,8 +451,11 @@ SUBSYSTEM_DEF(job)
 
 	if(!joined_late)
 		var/obj/effect/landmark/start/spawn_mark = null
+		var/occupation_name = rank
+		if(H.mind.role_alt_title != rank)
+			occupation_name = H.mind.role_alt_title
 		for(var/obj/effect/landmark/start/landmark in landmarks_list)
-			if((landmark.name == rank) && !(locate(/mob/living) in landmark.loc))
+			if((landmark.name == occupation_name) && !(locate(/mob/living) in landmark.loc))
```

Comment:
А если на станции нету лендмарки с таким же именем, как подпрофа чела? Его же заспавнит в жопе. Тебе тогда нужно для каждой подпрофы делать лендмарку

Reply:
я перепроверю щас, но у меня спавнило там, где ранк

Reply:
я не знаю как это работает так, но работает значит похуй
![11111](https://user-images.githubusercontent.com/96499407/189404908-1c40342c-cb44-43b2-8956-854979a2bad8.png)

Reply:
также с куком, с вирусологом и тд

Reply:
ответ не подходит. Тут либо я ни за что доебался, либо ты не понимаешь че делаешь.

Reply:
кто-то подумал за меня и сделал чтобы спавн не ломался при отсутствии лендмарки. Удобно.

Reply:
Я не вижу где это, я вижу что спавн будет где-то в жопе

Reply:
Изменений на 2 строчки. Ты сделал на локалке и оно у тебя сломалось? У меня какой-то особенный вскод который незаметно чинит код за меня? Или дриммейкер запускает поддельный билд? Я три раза с перерывом в день компилил и запускал. Проверил вирусолога, два раза инженера, роботеха, повара и 4 вида тестовиков, все спавнятся где должны.

Reply:
Вот щас ещё раз прийдётся переключать ветку и показывать

Reply:
![image](https://user-images.githubusercontent.com/26389417/189497407-4cc9d85f-aa2f-495f-b3c6-da6bed61a6ab.png)
Компилируем в голове.

1. var/occupation_name = rank (occupation_name = Assistent)
2. if(H.mind.role_alt_title != rank) (Паранормальщик != асситент = TRUE)
3. occupation_name = H.mind.role_alt_title (occupation_name = паранормальщик)
4. if((landmark.name == occupation_name)) (landmark.ассистент == паранормалищик = FALSE)
5. if(!spawn_mark)
			spawn_mark = locate("start*[rank]") // use old stype
			
			В итоге спавнимся в "start*[rank]"

Reply:
Мне не надо показывать, мне нужно по коду доказать

Reply:
Если ты хотел чтобы я объяснил что такое *start*[rank] то я не знаю откуда и что это.

Reply:
Я понял че это такое. Все подпрофы будут спавниться в одном тайле, так как тег, вроде как, может быть только 1 с 1 срокой.

Reply:
![1111111](https://user-images.githubusercontent.com/96499407/189497850-6a08f14f-bb61-47a3-9fb6-f74d1c68c949.png)

Reply:
> This may be assigned to a unique text string identifying a particular object. A reference to the object can then be obtained by using locate().

https://secure.byond.com/docs/ref/index.html#/datum/var/tag

Оно по докам только 1 лендмарку должно содержать